boost-common/src/main/java/io/openliberty/boost/common/config/BoostProperties.java

```diff
@@ -25,6 +25,8 @@
     public static final String DATASOURCE_PORT_NUMBER = "boost.db.portNumber";
     public static final String DATASOURCE_USER = "boost.db.user";
     public static final String DATASOURCE_PASSWORD = "boost.db.password";
+    public static final String PASSWORD_ENCRYPTION_METHOD = "boost.pkg.prop.encryption";
```

Comment:
Minor comment... we might want to change the name of the external property to something like "boost.password.encryption" or something like that. Some indicator that this is for password encryption?

Reply:
Ya, think it should still be related to the db password though? Maybe "boost.db.password.encryption"?

Reply:
Hmm good question. Depends if we want to have each type of password have a different encryption or if we want all passwords configured in boost to use the same type.  I'd think we want a generic "encrypt all passwords with this encoding" property, so maybe leave off the "db" part of the name.  But @scottkurz or @cherylking might have a better idea.